libcontainerd/pausemonitor_unix.go

```diff
@@ -1,3 +1,5 @@
+// +build linux solaris
```

Comment:
will revisit the build tags. thanks for your comments @vdemeester 
sorry about the delay in replying to them. Was on vacation earlier this week.
Will update the changeset shortly.